### ISSUE TITLE
Is it possible to open web-console when running tigris locally?

### ISSUE THREAD

**Issue by finalclass** [tigrisdata/tigris-cli#226]
How to open Data Explorer (a web-console) to explore my database when running locally?

**Comment by efirs** [comment]
We do not currently have webconsole support in local setup.
But you can explore you data using CLI.
For example to read all records from the collection:
```
tigris login local # to point CLI to local instance
tigris read {coll name} 
```

**Comment by finalclass** [comment]
Thx for a quick answer.
Local capabilities are very nice but sometimes it's useful to see thedata in some GUI.
Do you plan adding the webconsole to the local setup some day?

**Comment by ovaistariq** [comment]
Yes, we do plan on adding web console for local setup as well. In the meantime you can use Tigris cloud for local development as well.
